lettermgmt/models/letter_category.py

```diff
@@ -0,0 +1,15 @@
+# -*- encoding: utf-8 -*-
+# Copyright (C) 2009-Today Tech Receptives(http://techreceptives.com).
+# Parthiv Pate, Tech Receptives, Open Source For Ideas
+# Hardikgiri Goswami <hardikgiri.goswami@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields
+
+
+class LetterCategory(models.Model):
+    """ Class to define the category of letter like : classified,
+    confidential, personal, etc. """
+    _name = 'letter.category'
+    _description = "Letter Category"
+    name = fields.Char('Type', required=True)
```

Comment:
Maybe remove field string `'Type'`, since the field is supposed to be the name of the letter category, just `'Name'` makes sense.